index.bs

```diff
@@ -372,6 +378,136 @@ One example of a feature that should be limited to secure contexts
 is geolocation, since the authentication and confidentiality provided by
 secure contexts reduce the risks to user privacy.
 
+<h3 id="private-browsing-mode">Consider how your API should behave in private browsing mode</h3>
+
+In general,
+APIs should behave the same in and out of [private browsing mode](https://www.w3.org/2001/tag/doc/private-browsing-modes/).
+This is because
+the use of private browsing mode should not be revealed to sites.
+See [[#do-not-expose-use-of-private-browsing-mode]].
+
+However,
+there are some situations in which
+APIs should consider behaving differently
+when private browsing mode is engaged,
+as discussed in our [privacy & security questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/#private-browsing).
+
+For instance,
+if your API would reveal information
+that would allow for the correlation
+of a single user’s activity
+both in and out of private browsing mode,
+consider possible [mitigations](https://www.w3.org/TR/security-privacy-questionnaire/#mitigations) such as
+introducing noise
+or augmenting permission prompts with additional information
+which would help users meaningfully consent to such correlation
+(see [[#consent]]).
+
+<div class="example">
+
+The [Payment Request API](https://www.w3.org/TR/payment-request/)'s [show() method](https://www.w3.org/TR/payment-request/#show-method), when called,
+allows User Agents to <q cite="https://www.w3.org/TR/payment-request/#show-method">to act as if the user had immediately [aborted the payment request](https://www.w3.org/TR/payment-request/#dfn-user-aborts-the-payment-request)</q>.
+
+This enables User Agents to automatically abort payment requests
+in private browsing mode
+(thus protecting [sensitive information](https://www.w3.org/TR/security-privacy-questionnaire/#sensitive-data)
+such as the user's shipping or billing address)
+without revealing that private browsing mode is engaged.
+
+</div>
+
+Private browsing modes enable users to browse the web
+without leaving any trace of their private browsing on their device.
+Therefore, APIs which provide client-side storage
+should not persist data stored
+while private browsing mode is engaged
+after it is disengaged.
+
+<div class="example">
+  
+User Agents which support [localStorage](https://html.spec.whatwg.org/multipage/webstorage.html#dom-localstorage)
+should not persist storage area changes
+made while private browsing mode is engaged.
+
+If the User Agent has two simultaneous sessions with a site,
+one in private browsing mode and one not,
+storage area changes made in the private browsing mode session
+should not be revealed to the other browsing session.
```

Comment:
this is also true in reverse, right?  Maybe worth saying so?

Reply:
Also fixed in 2e715a6.